app/Resources/views/row/download_section.html.twig

```diff
@@ -41,53 +41,7 @@
-                                        </div>
-                                        <div class="minimize">
-                                            <a class="minimizebutton" href="#"><span><strong>&ndash;</strong></span></a>
-                                        </div>
-                                        <div class="zoom">
-                                            <a class="zoombutton" href="#"><span><strong>+</strong></span></a>
-                                        </div>
-                                    </div>
-                                    ezplatform@eZ-Systems: ~/www
-                                </div>
-                                <div class="content">
-                                    <div class="orange">
-                                        <p>&#36; curl -sS https://getcomposer.org/installer | php</p>
-                                        <p>&#36; php composer.phar create-project -h</p>
-                                    </div>
-
-                                    <div class="help">
-                                        <p>Usage:</p>
-                                        <p>create-project [options] [--] [&lt;package&gt;] [&lt;directory&gt;] [&lt;version&gt;]</p>
-
-                                        <p>
-                                            <span class="orange">&lt;package&gt;</span>: Distribution to install, ezsystems/ezplatform is a clean installs of eZ Platform
-                                        </p>
-                                        <p>
-                                            <span class="orange">&lt;directory&gt;</span>: Folder to extract to, if omitted same as package name so in example: ezplatform.
-                                        </p>
-                                        <p>
-                                            <span class="orange">&lt;version&gt;</span>: Optional, when omitted you'll get latest stable. Examples: <br>
-
-                                            ~1.3.0: To pick latests 1.3 release, to pick latests 1.x release use ~1.3.<br>
-                                            v1.1.0 : To pick a specific tag, could also have been v1.0.0-rc1<br>
-                                            dev-master : Picks a development version from a branch, here master.
-                                        </p>
-                                    </div>
-
-                                    <div class="orange">
-                                        <p>&#36; php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatform</p>
-                                        <p>&#36; cd ezplatform/</p>
-                                    </div>
-                                </div>
-                            </div>
+                            {% include "composer/install.html.twig" with {} only %}
```

Comment:
> I found it more readable to say I want to include with nothing only rather than just "only"

`only` means that you don't want to inject parent context. It's maybe a bit misleading but works very well. I would omit `with {}` anyway. This feature is pretty well explained in http://twig.sensiolabs.org/doc/tags/include.html :-)